back/app/uploaders/project_folders/file_uploader.rb

```diff
@@ -3,7 +3,8 @@
 module ProjectFolders
   class FileUploader < BaseFileUploader
     def extension_allowlist
-      %w[avi csv doc docx key mkv mp3 mp4 numbers odp ods odt pages pdf ppt pptx txt xls xlsx]
+      %w[avi csv doc docx key mkv mp3 mp4 numbers odp ods odt pages pdf ppt pptx txt xls xlsx] +
+        BaseImageUploader::ALLOWED_TYPES
```

Comment:
We seem to often override this with the same method content. Why?

Reply:
We don't now. I've removed overrides of `extension_whitelist` that were exactly the same as this method.